test-configs.yaml

```diff
@@ -146,6 +146,83 @@ test_plans:
             - 'defconfig+virtualvideo'
             - 'multi_v7_defconfig+virtualvideo'
 
+  uefi-arm64-gicv2:
+    rootfs: buildroot_ramdisk
+    params: {ctx_gicversion: "gic-version=2", ctx_cpu: "cortex-a57",
```

Comment:
There's no need to use curly brackets for a large dictionary, it's easier to read with the indented syntax:
```yaml
    params:
      ctx_gicversion: "gic-version=2"
      ctx_cpu: "cortex-a57",
      image_url_uefi: 'http://people.linaro.org/~ard.biesheuvel/kernelci/QEMU_EFI.fd-AARCH64-RELEASE-325ad6226099'
```